less/admin/ExtensionPage.less

```diff
@@ -0,0 +1,143 @@
+@extension-list-column-width: 410px;
+
+.ExtensionPage {
+  min-height: 110vh;
+
+  .ExtensionPage-header {
+    .ExtensionTitle {
+      margin: 20px 0 15px;
+    }
+    .helpText {
+      margin-bottom: 5px;
+    }
+
+    h2 {
+      margin-left: 35px;
+    }
+  }
+
+  .ExtensionPage-header,
+  .ExtensionPage-permissions-header {
+    background: @control-bg;
+
+    h2 {
+      color: @muted-color;
+
+      span {
+        font-size: 13px;
+        color: @muted-color;
+        font-weight: normal;
+      }
+    }
+
+    button {
+      i {
+        display: unset;
+      }
+    }
```

Comment:
best to use classes where possible, `.Button-icon` would work here

this is a good case of why the primary button weirdly hides the icon lol, something we'll need to fix in the future.